pages.it/linux/zypper.md

```diff
@@ -0,0 +1,23 @@
+# zypper
+
+> Sistema di Gestione dei Pacchetti di SUSE e openSUSE.
+
+- Sincronizza il database dei pacchetti e delle versioni disponibili:
```

Comment:
Oh we're having a discussion about this example description in another PR: https://github.com/tldr-pages/tldr/pull/6220#discussion_r667516852, but your translation is OK for now.